source/vibe/stream/openssl.d

```diff
@@ -469,6 +471,17 @@ final class OpenSSLContext : SSLContext {
 			SSL_CTX_set_cipher_list(m_ctx, toStringz(list));
 	}
 
+	/** Make up a context ID to assign to the SSL context.
+
+		This is very required when doing client cert authentication,
+		otherwise many connections will go aborted.
+	*/
+	void setContextID()
```

Comment:
And you know what, I totally forgot what I actually implemented.

You're right, this function doesn't need to be public.